src/test/store/profile-view.test.js

```diff
@@ -196,3 +201,205 @@ describe('expand all call node descendants', function() {
+        getState()
+      )
+    ).toMatchSnapshot();
+  });
+  it('matches the last stored run of selectedThreadSelector.getFilteredThread', function() {
+    expect(
+      selectedThreadSelectors.getFilteredThread(getState())
+    ).toMatchSnapshot();
+  });
+  it('matches the last stored run of selectedThreadSelector.getRangeSelectionFilteredThread', function() {
+    expect(
+      selectedThreadSelectors.getRangeSelectionFilteredThread(getState())
+    ).toMatchSnapshot();
+  });
+  it('matches the last stored run of selectedThreadSelector.getCallNodeInfo', function() {
+    expect(
+      selectedThreadSelectors.getCallNodeInfo(getState())
+    ).toMatchSnapshot();
+  });
+  it('matches the last stored run of selectedThreadSelector.getCallNodeMaxDepth', function() {
+    expect(selectedThreadSelectors.getCallNodeMaxDepth(getState())).toEqual(4);
+  });
+  it('matches the last stored run of selectedThreadSelector.getSelectedCallNodePath', function() {
+    expect(selectedThreadSelectors.getSelectedCallNodePath(getState())).toEqual(
+      [A, B]
+    );
+  });
+  it('matches the last stored run of selectedThreadSelector.getSelectedCallNodeIndex', function() {
+    expect(
+      selectedThreadSelectors.getSelectedCallNodeIndex(getState())
+    ).toEqual(1);
+  });
+  it('matches the last stored run of selectedThreadSelector.getExpandedCallNodePaths', function() {
+    expect(
+      selectedThreadSelectors.getExpandedCallNodePaths(getState())
+    ).toEqual([[A], [A, B], [A]]);
+  });
+  it('matches the last stored run of selectedThreadSelector.getExpandedCallNodeIndexes', function() {
+    expect(
+      selectedThreadSelectors.getExpandedCallNodeIndexes(getState())
+    ).toEqual([0, 1, 0]);
+  });
+  it('matches the last stored run of selectedThreadSelector.getCallTree', function() {
+    expect(selectedThreadSelectors.getCallTree(getState())).toMatchSnapshot();
+  });
+  it('matches the last stored run of selectedThreadSelector.getFlameGraphTiming', function() {
+    expect(
+      selectedThreadSelectors.getFlameGraphTiming(getState())
+    ).toMatchSnapshot();
+  });
+  it('matches the last stored run of selectedThreadSelector.getFriendlyThreadName', function() {
+    expect(selectedThreadSelectors.getFriendlyThreadName(getState())).toEqual(
+      'Thread with samples'
+    );
+  });
+  it('matches the last stored run of selectedThreadSelector.getThreadProcessDetails', function() {
+    expect(
+      selectedThreadSelectors.getThreadProcessDetails(getState())
+    ).toMatchSnapshot();
+  });
+  it('matches the last stored run of selectedThreadSelector.getSearchFilteredMarkers', function() {
+    expect(
+      selectedThreadSelectors.getSearchFilteredMarkers(getState())
+    ).toMatchSnapshot();
+  });
+  it('matches the last stored run of selectedThreadSelector.unfilteredSamplesRange', function() {
+    expect(selectedThreadSelectors.unfilteredSamplesRange(getState())).toEqual({
+      end: 9,
+      start: 0,
+    });
```

Comment:
in cases like this you might benefit from jest-in-case

but yea, looks great

Reply:
Thanks, I'm looking at jest-in-case for another PR.